x/xbasm/parse.go

```diff
@@ -39,3 +39,169 @@ func ParseFnCallClaims(v basm.MarshaledAttestedObject) (*FnCallClaims, error) {
 
 	return c, nil
 }
+
+func isNonZero(b []byte) bool {
+	isNonZero := func(b byte) bool { return b != 0 }
```

Comment:
somewhat of a strange variable name shadowing here of `isNonZero`, right? Could probably pass `func(b byte) bool { return b != 0 }` directly into the ContainsFunc, just because the whole function is so short and clearly named

Reply:
updated in delphi